
class MafiaError(Exception):
    pass

class MafiaUserError(Exception):
    pass
